components/threads/actions/post-chat-message/post-chat-message.mjs

```diff
@@ -0,0 +1,41 @@
+import threads from "../../threads.app.mjs";
+
+export default {
+  key: "threads-post-chat-message",
+  name: "Post a Chat Message",
+  description: "Post a message to a chat. First, make sure you add your Bot user to the chat.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    threads,
+    chatID: {
+      propDefinition: [
+        threads,
+        "chatID",
+      ],
+    },
+    body: {
+      propDefinition: [
+        threads,
+        "body",
+      ],
+      label: "Message",
+      description:
+        "This is the body of the message. To format the body, you can send markdown.",
```

Comment:
Compose your chat message here, and include markdown if you'd like. See [here](https://github.com/ThreadsHQ/api-documentation#2-1) for examples.